src/lib/util/RichDisplay.js

```diff
@@ -91,11 +91,11 @@ class RichDisplay {
 
 	/**
 	 * Sets emojis to a new set of emojis
-	 * @param {RichDisplayEmojisObject} emojis An object containing replacement emojis to use instead
+	 * @param {RichDisplayEmojisObject} emojis An object containing replacement emojis to use instead with the option to remove options that are not wanted
 	 * @returns {RichDisplay} this RichDisplay
 	 */
 	setEmojis(emojis) {
-		Object.assign(this.emojis, emojis);
+		this.emojis = emojis;
```

Comment:
The old code should stay here. Instead, make the jump an option exactly as stop is an option now.